src/applications/vaos/appointment-list/components/AppointmentsPage/ScheduleNewProjectCheetah.jsx

```diff
@@ -32,7 +32,7 @@ export default function ScheduleNewProjectCheetah({
         to="/new-project-cheetah-booking"
         onClick={startNewAppointmentFlow}
       >
-        Schedule COVID-19 vaccination
+        Learn more
```

Comment:
I think there are some other text changes in the screenshot in the issue.

Reply:
Oh great catch.